susePublicCloudInfoClient/lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -205,6 +220,11 @@ def __warn(str, out=sys.stdout):
     out.write("Warning: %s" % str)
 
 
+def __error(str, out=sys.stdout):
```

Comment:
Error should be written to stderr, since the method is called __error that should be the default